src/analysis/utils/tool-utils.ts

```diff
@@ -112,10 +114,13 @@ const Cromwell: AppTool = { label: 'CROMWELL', isPauseUnsupported: true };
 
 const HailBatch: AppTool = { label: 'HAIL_BATCH', isPauseUnsupported: true };
 
+const WDS: AppTool = { label: 'WDS', isPauseUnsupported: true };
```

Comment:
I assume `isPauseUnsupported` should be true here.

Reply:
I would say so too yes